docs/content/Contribution guidelines/operations/code-management.md

```diff
@@ -7,3 +7,128 @@
+
+git commit -m "your message"
+
+git push -u origin branch-name (-u sets upstream to origin which is your remote github repository)
+```
+### When you have completed your work and tested it locally, then you should perform a squash of the git commits to make the upcoming push request more manageable.
+
+To perform a squash, checkout the branch you want to squash,
+1. use the "git log" command to see the history of commits to the branch
+2. Count the number of commits you want to squash
+3. use the "git rebase -i HEAD~n" where n is the number of commits you would like to squash together. (There are other ways to do this)
+4. The text editor you have configured to use with git should automagically open your source and you will see a list of commits preceded by "pick". Leaving the first "pick" as it is, replace the remaining "pick"s with "squash"es. 
+5. Save the text file and exit the editor.
+6. The text editor will open again to let you edit comments for your new squashed commit.
+7. Make your edits if any and save and exit the file.
+The commits will then be squashed into one commit.
+8. run the "git push --force-with-lease" command
+
+### When you are done, push your changes to your remote branch
+```
+git push -u origin <branch-name>
+
+or 
+
+git push --force-with-lease
+```
+
+## Run Actions (automated workflow tests) manually in your personal Github repository
+1. Select the "Actions" tab towrd the upper left of your github personal web page. This will cause a list of Actions to show.
+2. Select the action you wish to execute from the list of Actions. For example you might chose "docs-ecutable - example1".
+Note: docs-ecutable should be described in a separate section. But in a nutshell it's a Continuous Integration automation technique of embedding scripts and data within the body of documentation, and then parsing and executing those scripts which in turn interpret and execute source code from a branch that you designate. It's somewhat similar to Travis. So the Action "docs-ecutable - example1" executes scripts and data embedded within the documentation for the Example 1 scenario, described in the Kubestellar documents. Those scripts will run using the source code pointed to by the next step, step 3.
+3. Select the source code branch you wish to exercise by
+  1. select the black and white "Run Workflow" on the right side of your github web page. This will open a dialog box.
+  2. within the dialog box, select the branch you wish to exercise by opening the dropdown labeled "use workflow from"
+  3. within the dialog box, select the green "Run Workflow" button 
+Your selected Action workflow will execute and the results will be available when it completes.  
+
+## Create a Pull Request (PR) from your Github repo branch in order to request review and approval from the Kubestellar team
+
+Take a look at https://github.com/kubestellar/kubestellar/blob/main/CONTRIBUTING.md
```

Comment:
A relative URL would be better than an absolute URL here.

Reply:
I'm not arguing, but why?

Reply:
As a general rule, relative references means that you are not hamstrung by dependency on the previous release.